### ISSUE TITLE
Repository links don't get added to a the list

### ISSUE THREAD

**Issue by h4nc** [custom-components/hacs#130]
## Version of HACS
HACS version: 0.8.1 
Home Assistant version: 0.94.3

## Describe the bug
When I add a repository like
https://github.com/kalkih/mini-media-player

the plugin opens and HACS installs it (www/community), but the repository doesn't add to a list in "settings". In fact there is not list, like seen in the examples.

The plugin also appears in the store (so it is searchable), but still no repository link in "settings"

I deleted my old plugins also copied this:
```
  - url: /community_plugin/mini-media-player/mini-media-player.js
    type: module
```
to my lovelace.yaml.

The cards work. But yeah no list. 

About the log below. I had to hit the install button two times, to make it finish, so I think the errors below have nothing to do with the missing list.

## Debug log
2019-06-19 14:17:37 [homeassistant.core] Error doing job: Fatal error on transport
2019-06-19 14:17:57 [custom_components.hacs.hacs] kalkih/mini-media-player - custom_components does not exist in the repository.
2019-06-19 14:17:57 [custom_components.hacs.hacs] kalkih/mini-media-player - Could not register.

**Comment by ludeeus** [comment]
It's missing in the list since it's a default repository (It's already in the store) :)

The "issue" I see here is that you are allowed to add it, HACS should tell you that it already exists.

**Comment by ludeeus** [comment]
Corrected in [0.9.0](https://github.com/custom-components/hacs/releases/0.9.0) 🎉
